src/vos/vos_pool.c

```diff
@@ -266,7 +266,7 @@ vos_blob_unmap_cb(uint64_t off, uint64_t cnt, void *data)
 	return rc;
 }
 
-static int pool_open(PMEMobjpool *ph, struct vos_pool_df *pool_df, uuid_t uuid,
+static int pool_open(PMEMobjpool *ph, struct vos_pool_df *pool_df,
```

Comment:
(style)  need consistent spacing around '*' (ctx:WxV)

Reply:
I'm not sure why this is an error.